scapy/layers/l2.py

```diff
@@ -408,41 +453,49 @@ class ARP(Packet):
                                          (val is None or valid_net6(val)))
                   ))),
             ],
-            StrFixedLenField("pdst", None, length_from=lambda pkt: pkt.plen),
```

Comment:
I think we're going to disable the mypy issues for this check (that requires casting from Any) on all files in `layers/` (that are not Core scapy), because it's going to become such a mess. Will get back to this